google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -98,7 +98,7 @@ appengine.deployment.status.deploying=Deploying to App Engine
 appengine.deployment.error.invalid.cloudsdk=Invalid Cloud SDK directory path.
 appengine.deployment.error.cancelled=Deployment process was cancelled.
 appengine.facet.converter.description=Deprecated Google App Engine facets will be replaced with the latest version
-appengine.run.server.name=App Engine Development Server
+appengine.run.server.name=Google App Engine Development Server
```

Comment:
ah yes I forgot about that. I'm totally in favor of sneaking in the word 'local' here.